optimum/onnxruntime/modeling_ort.py

```diff
@@ -105,49 +107,68 @@
 """
 
 
-@add_start_docstrings(
-    """
-    Base ORTModel class for implementing models using ONNX Runtime. The ORTModel implements generic methods for interacting
-    with the Hugging Face Hub as well as exporting vanilla transformers models to ONNX using `transformers.onnx` toolchain.
-    The ORTModel implements additionally generic methods for optimizing and quantizing Onnx models.
-    """,
-)
 class ORTModel(OptimizedModel):
-    base_model_prefix = "onnx_model"
+    """
+    Base class for implementing models using ONNX Runtime.
+
+    The ORTModel implements generic methods for interacting with the Hugging Face Hub as well as exporting vanilla
+    transformers models to ONNX using `optimum.exporters.onnx` toolchain.
+
+    Class attributes:
+        - model_type (`str`, *optional*, defaults to `"onnx_model"`) -- The name of the model type to use when
+        registering the ORTModel classes.
+        - auto_model_class (`Type`, *optional*, defaults to `AutoModel`) -- The "AutoModel" class to represented by the
+        current ORTModel class.
+
+    Common attributes:
+        - model (`ort.InferenceSession`) -- The ONNX Runtime InferenceSession that is running the model.
+        - config ([`~transformers.PretrainedConfig`] -- The configuration of the model.
+        - use_io_binding (`bool`, *optional*, defaults to `True`) -- Whether to use I/O bindings with **ONNX Runtime
+        with the CUDAExecutionProvider**, this can significantly speedup inference depending on the task.
+        - model_save_dir (`Optional[str]`, *optional*) -- The directory where the model exported to ONNX will be saved.
+        By defaults, if the loaded model is local, the directory where the original model will be used. Otherwise, the
+        cache directory is used.
+        - latest_model_name (`str`, *optional*, defaults to `"model.onnx"` -- The name of the last ONNX model file.
+        - providers (`List[str]) -- The list of execution providers available to ONNX Runtime.
+    """
+
+    model_type = "onnx_model"
```

Comment:
I would avoid using model_type as transformers config already use this for an other thing (the type of architecture)

Reply:
Yes, but this is exactly what it's about I think.

Reply:
Do you mean that the name in transformers is bad?

Reply:
I misunderstood